internal/download/download.go

```diff
@@ -371,15 +372,25 @@ func ResolveRemoteDocument(ctx context.Context, d workflow.Document, outPath str
 		token = os.Getenv(strings.ToUpper(envVar))
 	}
 
-	res, err := download.Fetch(d.Location.Resolve(), header, token)
+	var fileResponse *http.Response
+	var err error
+	fileResponse, err = download.Fetch(d.Location.Resolve(), header, token)
 	if err != nil {
-		return "", err
+		// Retry with bearer in case this is a github PAT and the user forgot
+		if env.IsGithubAction() && !strings.HasPrefix(token, "bearer") {
```

Comment:
A few considerations:
* This could be running in GitHub Actions, but pulling source from non-GitHub
* This could be running locally and still happen
* Might want to check for folded string case on bearer (e.g. check both bearer and Bearer), unless that's handled elsewhere already

Can we check for github.com (or similar in the location for detection?

Reply:
That's all doable. I'm not terribly concerned if we add bearer to something that's not a github PAT. Because we only get to this point if auth already failed

Reply:
But I can scope it down a bit using some of the things you listed.

Reply:
Updated